include/occa/scripts/tests/compiler.cpp

```diff
@@ -19,6 +19,10 @@ int main(int argc, char **argv) {
 #elif defined(__ICC) || defined(__INTEL_COMPILER)
   return OCCA_INTEL_VENDOR;
 
+// Clang also defines __GNUC__, so check for __clang__ first
```

Comment:
This was set at the bottom since other compilers are wrappers on `clang`. Maybe we should move the `GNU` checks even lower

Reply:
Tricky... moved the clang check to its original place and GNU below that.